src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Runtime/ProvideCorrectArgumentsToFormattingMethodsTests.cs

```diff
@@ -147,134 +264,361 @@ public class C
 {
     void Method()
     {
-        var s = String.Format(""{0} {1} {2} {3}"", new object[] {1, 2});
-        Console.Write(""{0} {1} {2} {3}"", new object[] {1, 2, 3, 4, 5});
-        Console.WriteLine(""{0} {1} {2} {3}"", new object[] {1, 2, 3, 4, 5});
+        {|#0:" + invocation + @"(""{0} {1}"", 1)|};
```

Comment:
Here I highlight all format indexes without associated args, right?

Reply:
That would be my preference

---
In reply to: [508701842](https://github.com/dotnet/roslyn-analyzers/pull/4226#discussion_r508701842) [](ancestors = 508701842)